src/components/Navigation/Nav.tsx

```diff
@@ -72,25 +72,18 @@ export const Nav = ({ className }: NavProps) => {
       </a>
     ));
 
-  const handleClick = () => setIsOpen(!isOpen);
+  const handleClick = () => {
+    if (isMobile) {
+      document.body.style.overflow = isOpen ? 'auto' : 'hidden';
+    }
+    setIsOpen(!isOpen);
```

Comment:
Se que esto estaba desde antes, pero se podría cambiar por esto?
```suggestion
    setIsOpen(prev => !prev);
```

Más que nada para evitar errores en caso de que se realicen varios clicks seguidos y no quede bien actualizado el state, no sé que opinan o si tienen otra información sobre esto!

Reply:
a mí me parece correcto, no sé qué opina @Zyruks pero yo también suelo trabajar con el prev!

Reply:
Para mi en este caso es irrelevante, no va a cambiar nada

Reply:
Hasta donde yo se, practicamente es lo mismo no? igual puedo cambiarlo o dejarlo tal como esta, que opinan @afordigital @Zyruks

Reply:
Aunque ambas formas funcionarían en este contexto, recomendaría usar `setIsOpen(prev => prev)` por ser más segura y prevenir posibles comportamientos inesperados en actualizaciones de estado futuras.

Es una buena práctica para mantener nuestro código robusto y fácil de mantener.